site/content/agent-app-widgets/widget-sdk.md

```diff
@@ -0,0 +1,134 @@
+---
+weight: 45
+---
+
+# Widget SDK
+
+To use the JavaScript API you should import the [Agent App Widget SDK](https://www.npmjs.com/package/@livechat/agent-app-widget-sdk). 
+
+**Please remember that it's considered [deprecated](#developing-your-own-widget) and you should use the [Agent App SDK](#agent-app-sdk) instead.**
```

Comment:
I'd move it to the first paragraph. I'd even consider not including "Widget SDK" section at all.

Reply:
✅, although I chose not to remove the section. Since the vast majority of our customers rely on the older version of the SDK, removing this would leave them without any reference. Let's leave it for now and wait until the new SDK is fairly adopted.

Reply:
👍